### DISCUSSION TITLE
hide title in v.37

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#13942] [Q&A] [answered]
Hi, in mpv v.36 I could hide the title or at least change it to "." for a minimalistic look. Inside script-opts/osc.conf I put the line `title=.`
Now in v.37 it is back. How to tackle this?
`title-bar=no` doesn't help here.
Thanks for any help :)

v.37
![title](https://github.com/mpv-player/mpv/assets/139760266/6a99dc1c-fffc-44f1-ac0c-1d22d5cc4d15)

v.36
![36 title](https://github.com/mpv-player/mpv/assets/139760266/8874d2c8-1b6f-474f-bd2d-d9d8eb19e98f)

**Comment by na-na-hi** [comment]
`windowcontrols_title` and `title` are now separate options.

**Reply by geextahslex** [reply]
Do you know how to turn both off? Or set them to nothing, or something like that?

**Reply by na-na-hi** [reply]
Just set both of them to a space: 

```
windowcontrols_title= 
title= 
```

**Reply by geextahslex** [reply]
When I do that, then the title changes to "mpv"
![gfgffggf](https://github.com/mpv-player/mpv/assets/139760266/be5ab4a2-844d-4abc-83c0-e75cf51951b8)

**Reply by na-na-hi** [reply, marked as answer]
You didn't insert a space character after the =.

**Reply by geextahslex** [reply]
Awesome, thank you :)
